File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n = len(nums)
        lst = [0 for i in range(2*n)]
        for i in range(len(nums)):
            lst[i] = lst[i+n] = nums[i]
        return lst

File: Data Structures & Algorithms/majority-element/submission-0.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        int cand=-1, cnt=0;
        for(int i=0; i<nums.size(); i++){
            if(cand==-1 || cand == nums[i]){
                cand = nums[i];
                cnt++;
            }
            else cnt--;
            if(cnt<0) {
                cand = nums[i];
                cnt++;
            }
        }
        return cand;
    }
};